### PR TITLE
add: turborepo support

### PR DESCRIPTION
**Pull request by alimehasin**

### CODE REVIEW

File: kondo-lib/src/lib.rs [hunk 1]
```diff
@@ -27,6 +27,7 @@ const FILE_BUILD_ZIG: &str = "build.zig";
 const FILE_GODOT_4_PROJECT: &str = "project.godot";
 const FILE_CSPROJ_SUFFIX: &str = ".csproj";
 const FILE_FSPROJ_SUFFIX: &str = ".fsproj";
+const FILE_PROJECT_TURBOREPO: &str = "trubo.zig";
```
Comment (tbillington) [inline kondo-lib/src/lib.rs:30]: should this be `turbo.zig`?

Comment (alimehasin) [reply kondo-lib/src/lib.rs:30]: Ohh no it is my fault

Comment (alimehasin) [reply kondo-lib/src/lib.rs:30]: I will fix it rn

**Comment by tbillington** [comment]
Thanks for your contribution :)

**Pull request merged**